test/Microsoft.Azure.Devices.Edge.Test/SetupFixture.cs

```diff
@@ -42,6 +43,17 @@ public async Task BeforeAllAsync()
 
                     // Install IoT Edge, and do some basic configuration
                     await this.daemon.UninstallAsync(token);
+
+                    // Delete directories used by previous installs.
+                    List<string> directories = new List<string>() { "/run/iotedge", "/var/lib/iotedge", "/var/run/iotedge" };
```

Comment:
These paths are only true for `release/1.1` branch. If you want to be inclusive to 1.2, the `var/lib/aziot/*`, `/var/run/aziot/*`, `/etc/aziot` are needed to be removed as well.

Reply:
I believe that is handled is 1.2 and master branches and the intention is to keep this change isolated to only 1.1 branch and not merge it back to master